e2e/tests/init_test.go

```diff
@@ -242,8 +242,12 @@ func (s *suite) TheSessionForApplicationAndServiceContainsTheSupplier(appName st
 	s.Fatalf("session for app %s and service %s does not contain supplier %s", appName, serviceId, supplierName)
 }
 
-func (s *suite) TheApplicationSendsTheSupplierARelayRequestForService(appName string, supplierName string, requestName string, serviceId string) {
-	// TODO(#126, @Olshansk): Implement this step
+func (s *suite) TheApplicationSendsTheSupplierARequestForServiceWithData(appName, supplierName, serviceId, requestData string) {
+	res, err := s.pocketd.RunCurl("", requestData)
```

Comment:
We want to make sure we append `/{service}` to the end of the rpcURL when we get there